src/containers/SearchPage/components/SearchHeader.tsx

```diff
@@ -121,81 +107,72 @@ const SearchHeader = ({
-                {t("searchPage.resultType.searchPhraseSuggestion")}
-                {/* TODO: Check if we should include an option for link variant to remove all padding */}
-                <Button variant="link" onClick={() => handleSearchParamsChange({ query: suggestion })}>
-                  [{suggestion}]
-                </Button>
-              </Text>
-            )}
-          </div>
-        )}
-        {loading && <div aria-label={t("loading")} />}
+      <Heading>{t("searchPage.title")}</Heading>
+      <div>
+        <form action="/search/" onSubmit={handleSearchSubmit}>
+          <StyledSearchWrapper>
+            <InputContainer>
+              <Input
+                ref={inputRef}
+                type="search"
+                autoComplete="off"
+                id="search"
+                name="search"
+                placeholder={t("searchPage.searchFieldPlaceholder")}
+                value={searchValue}
+                onChange={(e) => onSearchValueChange(e.target.value)}
+              />
+              {searchValue && (
+                <IconButton
+                  variant="clear"
+                  aria-label={t("welcomePage.resetSearch")}
+                  value={t("welcomePage.resetSearch")}
+                  onClick={() => {
+                    onSearchValueChange("");
+                    inputRef.current?.focus();
+                  }}
+                >
+                  <Cross />
+                </IconButton>
+              )}
+            </InputContainer>
+            <IconButton
+              variant="primary"
+              type="submit"
+              aria-label={t("searchPage.search")}
+              title={t("searchPage.search")}
+            >
+              <Search />
+            </IconButton>
+          </StyledSearchWrapper>
+        </form>
+        <StyledHitsWrapper aria-live="assertive">
+          {!loading && query && (
+            <div>
+              {noResults ? (
+                <Text textStyle="label.small">
+                  {t("searchPage.noHitsShort", { query: query })}
+                  {activeSubjectFilters.length ? `. ${t("searchPage.removeFilterSuggestion")}` : undefined}
+                </Text>
+              ) : (
+                <Text textStyle="label.small">
+                  {t("searchPage.resultType.showingSearchPhrase")} &ldquo;{query}&rdquo;
+                </Text>
+              )}
+              {suggestion && (
+                <Text textStyle="label.small">
+                  {t("searchPage.resultType.searchPhraseSuggestion")}
+                  {/* TODO: Check if we should include an option for link variant to remove all padding */}
+                  <Button variant="link" onClick={() => handleSearchParamsChange({ query: suggestion })}>
+                    [{suggestion}]
+                  </Button>
+                </Text>
+              )}
+            </div>
+          )}
+          {loading && <div aria-label={t("loading")} />}
```

Comment:
Denne kan vi vel bare legge rett på spinner?

Reply:
Jeg var litt usikker på tanken bak den div-en i det hele tatt .. Legge på hvilken spinner?

Reply:
Tipper at tanken er at skjermleser skal si "Laster" når siden laster, men det kan vi vel få til med en aria-label inne i en live-region

Reply:
Ja det er jo greit nok, men hvorfor det ikke holder med den som ligger i SearchContainer lurte jeg litt på .. 🤔

Reply:
To forskjellige spinners er nok litt dumt, ja. Du får velge en av de!

Reply:
Eller enda greiere: Flytt loading-logikken lenger opp?

Reply:
lenger opp?